util/execdetails.go

```diff
@@ -330,3 +331,16 @@ func (td *TimeDetail) MergeFromTimeDetail(timeDetail *kvrpcpb.TimeDetail) {
 		td.KvReadWallTimeMs += time.Duration(timeDetail.KvReadWallTimeMs) * time.Millisecond
 	}
 }
+
+// ResolveLockDetail contains the resolve lock detail information.
+type ResolveLockDetail struct {
+	*RequestSource
```

Comment:
It looks a bit weird to include this in `xxxDetail`. Can we just pass it by `Context`?